nb2workflow/deploy.py

```diff
@@ -519,8 +580,17 @@ def deploy(git_origin,
                                 nb2wversion=nb2wversion)
     
     if local:
+        env_params = []
+        for name, resource in container['resources'].items():
+            for env in resource['env_vars']:
+                env_val = os.getenv(env)
+                if env_val:
+                    env_params += ["-e", f"{env}={env_val}"]
```

Comment:
You may just use the `-e env_name` form. Then the value is not exposed in `ps` etc.

Reply:
done